elastic/datadog_checks/elastic/elastic.py

```diff
@@ -235,13 +253,17 @@ def _get_urls(self, version):
 
         return health_url, stats_url, pshard_stats_url, pending_tasks_url, slm_url
 
-    def _get_data(self, url, send_sc=True):
+    def _get_data(self, url, send_sc=True, data=None):
         """
         Hit a given URL and return the parsed json
         """
         resp = None
         try:
-            resp = self.http.get(url)
+            if data:
+                headers = {'content-type': 'application/json'}
```

Comment:
Is this needed?

Reply:
Originally I needed it when testing, although now the tests work fine without manually adding `header`, so I will remove it.